doc/source/autodoc.rst

```diff
@@ -72,3 +77,45 @@ Example
       portName3 : in bit    -- This is a documentation comment for portName3
     );
     end entity entityName;
+
+Example with groups
+-------------------
+
+Below shows how to work with port groups and generics.
+See :ref:`group_example_doc` for how this code will look
```

Comment:
The example below shows how to work with port groups and generics.
See the output in :ref:`group_example_doc`.